docs/commands/subLoginTokenV4.md

```diff
@@ -0,0 +1,117 @@
+### subLoginToken: Establishes a subconnection to Exasol using an OpenID token
+
+This command invokes the login process, which establishes a
+subconnection between the client and Exasol. Using subconnections,
+the user can interact with Exasol in parallel using various commands.
+
+:warning: This command requires a TLS connection (i.e., `wss://...`).
+
+The login process is composed of four steps:
+
+1. The client sends the `subLoginToken` command including the requested protocol
+   version.
+   
+   Request fields:
+     * command (string) => command name: "subLoginToken"
+     * protocolVersion (number) => requested WebSocket protocol version, (e.g., 4)
+   
+   Request JSON format
+   ```javascript
+    {
+        "command": "subLoginToken",
+        "protocolVersion": <number>
+    }
+   ```
+
+
+2. The server responds with either
+     * "ok", in which case the login process continues in step 3, or
+     * "error", in which case the login process is aborted.
+
+   Response fields:
+     * status (string) => command status: "ok" or "error"
+     * exception (object, optional) => only present if status is "error"
+       * text (string) => exception message which provides error details
+       * sqlCode (string) => five-character exception code if known, otherwise "00000"
+   
+   Response JSON format
+   ```javascript
+    {
+        "status": <"ok" | "error">,
+        // if status is "error"
+        "exception": {
+                "text": <string>,
+                "sqlCode": <string>
+        }
+    }
+   ```
+   
+
+3. The client sends either an OpenID `accessToken` or a `refreshToken` and optionally other client information.
+
+   Request fields:
+     * accessToken (string, optional) => OpenID access token to use for the login process
+     * refreshToken (string, optional) => OpenID refresh token to use for the login process
+     * token (number) => token required for subconnection logins (see, EnterParallel)
```

Comment:
Can we update the EnterParallel to requestParallelConnectionsV4?

Reply:
Ah, good find. Fixed.